lib/crypto/base36.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
I'm a strong advocate for 'use strict' in all new modules. Not sure whether other people agree or not, but wanted to raise it.